img/i/invoice.2go.com.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" id="Layer_1" data-name="Layer 1" version="1.1" width="186.144" height="186.144"><defs id="defs7039"><style id="style7037"></style></defs><path d="M183.585 20.416L1.165 92.956a1.86 1.86 0 00.05 3.47l54.16 19.61a1.85 1.85 0 001.74-.25l102.68-74.81a.25.25 0 01.31.39l-90.92 79.07a1.85 1.85 0 00-.65 1.41v42.17a1.85 1.85 0 003.29 1.17l26.24-32.35a1.85 1.85 0 012.07-.57l33.47 12.13a1.85 1.85 0 002.34-1l50.05-120.55a1.85 1.85 0 00-2.4-2.44" id="path7043" fill="#00de8c" fill-rule="evenodd"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 124.2 25.451" width="25.451" height="25.451"><path d="M122.5-35.712.8 12.688c-.2.1-.4.2-.6.5-.1.2-.2.4-.2.7 0 .3.1.5.2.7.1.2.3.4.6.5l36.1 13.1c.2.1.4.1.6.1.2 0 .4-.1.6-.2l68.5-49.9h.1s.1 0 .1.1v.2l-60.6 52.5c-.1.1-.3.3-.3.4-.1.2-.1.4-.1.5v28.1c0 .3.1.5.2.7.1.2.4.4.6.5.2.1.5.1.8 0 .2-.1.5-.2.6-.4l17.5-21.6c.2-.2.4-.3.6-.4.2-.1.5-.1.8 0l22.3 8.1c.3.1.6.1.9 0 .3-.1.5-.4.6-.7l33.4-80.4c.1-.2.1-.5.1-.7 0-.2-.2-.5-.3-.6-.2-.2-.4-.3-.6-.3-.3 0-.6 0-.8.1" style="clip-rule:evenodd;fill:#324bff;fill-rule:evenodd"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 124.2 25.45"><path fill="#324bff" d="M122.5-35.71.8 12.69c-.2.1-.4.2-.6.5-.1.2-.2.4-.2.7 0 .3.1.5.2.7.1.2.3.4.6.5l36.1 13.1c.2.1.4.1.6.1.2 0 .4-.1.6-.2l68.5-49.9h.1s.1 0 .1.1v.2l-60.6 52.5c-.1.1-.3.3-.3.4-.1.2-.1.4-.1.5v28.1c0 .3.1.5.2.7.1.2.4.4.6.5.2.1.5.1.8 0 .2-.1.5-.2.6-.4l17.5-21.6c.2-.2.4-.3.6-.4.2-.1.5-.1.8 0l22.3 8.1c.3.1.6.1.9 0 .3-.1.5-.4.6-.7l33.4-80.4c.1-.2.1-.5.1-.7 0-.2-.2-.5-.3-.6-.2-.2-.4-.3-.6-.3-.3 0-.6 0-.8.1"/></svg>
```

Reply:
This does not display correctly for me.

Reply:
Argh, here's a working one:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25.45 19.89"><path fill="#324bff" d="M25.1.02.16 9.94c-.04.02-.08.04-.12.1a.3.3 0 0 0-.04.14c0 .07.02.1.04.15.02.04.06.08.12.1l7.4 2.68c.04.03.08.03.12.03a.3.3 0 0 0 .13-.05L21.84 2.87h.02s.03 0 .03.02v.04L9.47 13.69c-.02.02-.06.06-.06.08-.02.04-.02.08-.02.1v5.76c0 .06.02.1.04.14.02.05.08.09.12.1.04.03.1.03.16 0 .04-.01.1-.03.13-.07l3.58-4.43a.43.43 0 0 1 .13-.08c.04-.02.1-.02.16 0l4.57 1.66c.06.02.12.02.18 0s.1-.09.13-.15L25.43.33c.02-.04.02-.1.02-.15 0-.04-.04-.1-.06-.12-.04-.04-.08-.06-.12-.06-.06 0-.13 0-.17.02"/></svg>
```

Reply:
Never mind. my suggestion is larger than your original one.